backend/websocketConnect.py

```diff
@@ -0,0 +1,26 @@
+from fastapi import FastAPI, WebSocket, WebSocketDisconnect
```

Comment:
Code Structure:
The WebSocket endpoint is clear and straightforward.
The use of try-except blocks for handling connection acceptance and communication errors is good practice. It adds robustness to the connection management.

Error Handling:
Consider adding a specific message for why the WebSocket connection might fail. This can help users debug more effectively.

Reply:
Lines 13, and 25 do this, the {e} is the error message that tells the user what went wrong